Makefile

```diff
@@ -1,24 +1,10 @@
 SHELL := /bin/bash
 
-validation-all:  ## Run Validation on all Endpoints
-	@echo ">> running validations for all endpoints .."
-	./run-validations.sh
+validate: ## Validate a given endpoint request or response
+	@node compiler/run-validations.js --api $(api) --type $(type) --stack-version $(stack-version)
```

Comment:
```suggestion
  test -n "$(stack-version)" # missing stack-version param
	@node compiler/run-validations.js --api $(api) --type $(type) --stack-version $(stack-version)
```

Reply:
downstream check makes this irrelevant

Reply:
The script runs the validation of the parameters.